src/schism/schism_esmf_util.F90

```diff
@@ -2524,9 +2547,9 @@ subroutine SCHISM_StateImportWaveTensor(state, isPtr, rc)
   type(ESMF_StateItem_Flag)  :: itemType
 
   real(ESMF_KIND_R8), pointer :: farrayPtr1(:) => null()
-  real(ESMF_KIND_R8), pointer :: eastward_wave_radiation_stress(:) => null()
-  real(ESMF_KIND_R8), pointer :: eastward_northward_wave_radiation_stress(:) => null()
-  real(ESMF_KIND_R8), pointer :: northward_wave_radiation_stress(:) => null()
+  real(ESMF_KIND_R8), allocatable :: eastward_wave_radiation_stress(:)
```

Comment:
Is there a reason to prefer allocatables over pointers?

Reply:
I am not sure why but pointers was not working and creating issue. So, since this is local and those are deallocated, I think it is safe to go with this way. We might also consider to make them module variables. I don't think it will create performance issue but we might get rid of allocating and deallocating in every time.

Reply:
Thanks for your answer.  I don't think the allocate/deallcoate carries a huge performance cost, as it is only called once per exchange.   I would *not* add more module variables as that costs memory permanently. 
I don't see through the intricacies of pointer vs allocatable and have just for myself decided to use mostly pointers.  But allocatable are fine, too.